sampler/stories/default/Spinner.js

```diff
@@ -64,7 +64,7 @@ storiesOf('Agate', module)
 						centered={boolean('centered', Config)}
 						color={select('color', ['dark', 'light'], Config, 'light')}
 						scrim={boolean('scrim', Config)}
-						size={select('size', ['medium', 'small'], Config)}
+						size={select('size', ['huge', 'medium', 'small'], Config)}
```

Comment:
We'll likely need `'smallest'` in this list as well, to correlate with the sizes of `Icon.